### DISCUSSION TITLE
MTEX 6.0.beta1

### DISCUSSION THREAD

**Discussion by ralfHielscher** [mtex-toolbox/mtex#1947] [Announcements]
Dear MTEX community,

we are very proud to announce [MTEX 6.0.beta1](https://mtex-toolbox.github.io/download) which is a major step forward as it come with 3d support for EBSD and grain maps and hence allows to analyze and compare data taken from different directions of the same sample. Furthermore, we have included an interface to Neper and the ability align spherical plots e.g., pole figures arbitrarily in the plotting plane.
More information can be found in the [changelog](https://mtex-toolbox.github.io/changelog#1).

We would be very grateful if you could help us testing the new functionality. We hope you enjoy MTEX and would be happy to welcome you at the [MTEX workshop 2024](https://mtex-toolbox.github.io/workshop24).

The [MTEX team](https://mtex-toolbox.github.io/people).

**Comment by nyyssont** [comment]
This is a wonderful development! Exactly the right direction for MTEX in my opinion. I look forward to testing the new capabilities.

**Comment by Tijmenvermeij** [comment]
Very nice! Looking forward to trying it out 😁
